lib/main.dart

```diff
@@ -11,7 +11,7 @@ import 'constants/supabase.dart';
 Future<void> main() async {
   WidgetsFlutterBinding.ensureInitialized();
 
-  await dotenv.load(fileName: ".env");
+  await dotenv.load(fileName: 'lib/.env');
```

Comment:
```suggestion
  await dotenv.load(fileName: ".env");
```
We want this in the root directory